_pages/welcome-to-TTS/classes/intro-to-product-and-open-source.md

```diff
@@ -39,7 +39,7 @@ The last 5-6 years have seen a slow but steady growth in the government in the o
 - Using other open source projects
 - Contributing and being involved in open source communities
 
-In 2016 the White House published a governmentwide open source policy, the [Federal Source Code Policy](https://sourcecode.cio.gov). We helped with that work and with the creation of [code.gov](https://www.code.gov), the accompanying code sharing platform. We have our own [strong open source policy](/open-source) and have also brought together in one place [many resources that agencies find useful](http://pages.18f.gov/open-source-program/).
+In 2016, the White House published a government-wide open source policy: the [Federal Source Code Policy](https://sourcecode.cio.gov). 18F helped with that work and with the creation of [code.gov](https://www.code.gov), the accompanying code sharing platform. We have our own [strong open source policy](/open-source) and have also brought together in one place [many resources that agencies find useful](http://pages.18f.gov/open-source-program/).
```

Comment:
1) The _AP Style Guide_ supports `governmentwide` (same pattern as `nationwide` and `worldwide`), no? It's possible 18F has settled on something different, but the [content guide](https://content-guide.18f.gov/specific-words-and-phrases/) doesn't appear to address this.
2) Great clarification, thank you! 😅